ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WSQueryEndpointTest.java

```diff
@@ -88,6 +90,11 @@
   private Query query;
   private WSQueryEndpoint wsQueryEndpoint;
 
+  @BeforeClass
```

Comment:
Does the websocket end point support the request offset?  If it does, its worth adding a test. If not, then it should reject any request that contains an offset.